.github/workflows/ci.yml

```diff
@@ -1,36 +1,25 @@
-name: Test
-
+name: CI
 on:
   push:
-    branches: [ master ]
+    tags:
+      - v*
+    branches:
+      - master
```

Comment:
Should we allow CI to run when not submitted to master?

Reply:
The CI is allow for all PR, regardless of the branch --- that is how its running on this PR --- but only on pushes to master branch.